packages/shopify-codemod/test/fixtures/split-return-assignments/multi.output.js

```diff
@@ -0,0 +1,14 @@
+foo(() => {
+  console.log(bar);
+  this.sho = this.doo;
+  this.doo = this.zoo;
+  this.zoo = 0;
+  return this.sho;
+});
+
+foo(() => {
+  foo();
+  foo = bar;
+  bar = baz;
```

Comment:
I still lean to just moving the whole assignment up rather than splitting it into individual assignments. I'm not 100% sold on that, though — what do you think?